ocaml/idl/datamodel.ml

```diff
@@ -8994,6 +8994,7 @@ let pgpu =
       field ~qualifier:DynamicRO ~ty:(Map (Ref _vgpu_type, Int)) ~lifecycle:[Published, rel_vgpu_productisation, ""] ~default_value:(Some (VMap [])) "supported_VGPU_max_capacities" "A map relating each VGPU type supported on this GPU to the maximum number of VGPUs of that type which can run simultaneously on this GPU";
       field ~qualifier:DynamicRO ~ty:(pgpu_dom0_access) ~lifecycle:[Published, rel_cream, ""] ~default_value:(Some (VEnum "enabled")) "dom0_access" "The accessibility of this device from dom0";
       field ~qualifier:DynamicRO ~ty:Bool ~lifecycle:[Published, rel_cream, ""] ~default_value:(Some (VBool false)) "is_system_display_device" "Is this device the system display device";
+      field ~qualifier:DynamicRO ~ty:(Map (String,String)) ~lifecycle:[Prototyped, rel_vgpu_migration_tech_preview, ""] ~default_value:(Some (VMap [])) "compatibility" "PGPU metadata to determine whether a VGPU can migrate between two PGPUs";
```

Comment:
I think that a more specific `compatibility_metadata` would be more specific and more directly related to the `pgpu_metadata` type that we used elsewhere